### DISCUSSION TITLE
是否支持iOS平台

### DISCUSSION THREAD

**Discussion by arror** [xjasonlyu/tun2socks#114] [Q&A] [answered]
请问大佬是否支持iOS平台，如果未明确支持，是否有相关计划？

目前在寻找go-tun2socks的替代方案。

**Comment by xjasonlyu** [comment, marked as answer]
你可以用gomobile试试

**Reply by arror** [reply]
经测试是可以的

https://github.com/Clash-for-Apple/clash-interface/blob/main/clash/tun.go
https://github.com/Clash-for-Apple/clash-interface/blob/main/clash/tun2socks.go

**Comment by ding2chen** [comment]
@Arror 请问你是如何在ios上使用的呢
